pylint_plugins/BUILD

```diff
@@ -8,6 +8,7 @@ python_sources()
 
 python_tests(
     name="tests",
+    tags=["unit"],
```

Comment:
This allowed me to run all unit tests without explicitly running the pylint_plugins tests.